app/configurator/components/layout-configurator.tsx

```diff
@@ -187,49 +204,50 @@ const LayoutSharedFiltersConfigurator = () => {
             {potentialSharedFilters.map((filter) => {
               const dimension = dimensionsByIri[filter.componentIri];
               const sharedFilter = sharedFiltersByIri[filter.componentIri];
-              console.log(
-                dimension,
-                sharedFilter,
-                filter.componentIri,
-                sharedFiltersByIri
-              );
+
               if (!dimension || !isTemporalDimension(dimension)) {
                 return null;
               }
               return (
-                <Box
-                  display="flex"
-                  alignItems="center"
-                  justifyContent="space-between"
-                  width="100%"
-                  key={filter.componentIri}
-                >
-                  <Typography variant="body2" flexGrow={1}>
-                    {dimension.label || filter.componentIri}
-                  </Typography>
-                  <FormControlLabel
-                    sx={{ mr: 0 }}
-                    labelPlacement="start"
-                    disableTypography
-                    label={
-                      <Typography variant="body2">
-                        <Trans id="controls.section.shared-filters.shared-switch">
-                          Shared
-                        </Trans>
-                      </Typography>
-                    }
-                    control={
-                      <Switch
-                        checked={!!sharedFilter ?? false}
-                        onChange={handleToggle}
-                        inputProps={{
-                          // @ts-expect-error ts(2322) - data-component-iri is not considered a valid attribute, while it is
-                          "data-component-iri": filter.componentIri,
-                        }}
-                      />
-                    }
+                <>
+                  <Box
+                    display="flex"
+                    alignItems="center"
+                    justifyContent="space-between"
+                    width="100%"
+                    key={filter.componentIri}
+                  >
+                    <Typography variant="body2" flexGrow={1}>
+                      {dimension.label || filter.componentIri}
+                    </Typography>
+                    <FormControlLabel
+                      sx={{ mr: 0 }}
+                      labelPlacement="start"
+                      disableTypography
+                      label={
+                        <Typography variant="body2">
+                          <Trans id="controls.section.shared-filters.shared-switch">
+                            Shared
+                          </Trans>
+                        </Typography>
+                      }
+                      control={
+                        <Switch
+                          checked={!!sharedFilter ?? false}
```

Comment:
Nit: I think you don't need `?? false` since the value is already casted to boolean?